### DISCUSSION TITLE
How to disable preview of every file in the sidebar?

### DISCUSSION THREAD

**Discussion by Bluebuster116** [sumatrapdfreader/sumatrapdf#3633] [Q&A]
Hello. Everytime whenever I try to open a file from Sumatra's own open file option in the menu bar just triggers a left sidebar with a list of every file in a given folder.

Because I am using an underpowered laptop the process is time consuming because I have so many different files among those which arent supposed to be read by Sumatra.

Is there a way to disable this feature or something?. The same happens when I try right click on Windows open with certain program dialog box on explorer.

My current version of SumatraPDF is 3.2 for 32 bit which by the way I am running it in Windows 7 Starter. Can someone help me please?.

**Comment by GitHubRulesOK** [comment]
That is unusual behaviour for 3.2 (there have been similar comments for some cases in 3.5 pre-release, but not afaik 3.2) 

I can happen when the "Folder" is selected and contains just images (which could also apply for a "ZIP" folder)

I suggest it may be the installation is corrupted, as it will not usually be a problem with a portable version, If I run 3.2 official portable then I only get the one file selected, unless I intentionally open a folder by say drag and drop.

I would suggest 
1) back up the settings file, by open advanced settings and save to a different folder.
2) then **without** SumatraPDF active delete the source setting file and try restart SumatraPDF which will start with a new settings file
If that clears the problem great (modify new settings with any content you need from old settings) but if not, then try re install
Using a clean version https://www.sumatrapdfreader.org//dl//rel/3.2/SumatraPDF-3.2-install.exe
for portable 3.2 use https://www.sumatrapdfreader.org//dl//rel/3.2/SumatraPDF-3.2.zip

**Reply by Bluebuster116** [reply]
I will describe the issue with more details because from your answer it appears to be a problem specific to my SumatraPDF 3.2 version which I mentioned above is for 32 bit.

I doubt that it is related with the installation as it was done cleanly.

When I use the program's menu option to open a file, lets say a pdf, sumatra opens the file in one tab but it also opens an additional tab with a left side bar showing all the files in the same folder from which the pdf came from and opening the first file in that folder.

The same happens in windows explorer by doing right click on the file and then selecting open with a certain program option. 

Maybe can you check if this is also happening to you?

Again, I don't know if there is a way in Sumatra settings for disabling this so it doesn't happen and to reassure that it is not doing this in the background.

I have the say that since I installed SumatraPDF it was not my default document reader. Documents were assigned to be opened by Google Chrome. I left it as it is because when I double click on a certain file it let me choose which program to open a certain file and then I could select Sumatra and this avoided the opening of that annoying tab.

In my folder, I have all kinds of files including office documents, text files, images and all of that.

I don't know the reason why SumatraPDF does open an additional tab with a left side bar. As I mentioned it takes much time by doing a preview of every other file in that folder.

Is there a solution which doesn't require reinstalling the program that I could try?

The portable app version is not for me as what I am looking is a more permanent solution.

**Reply by GitHubRulesOK** [reply]
Much of the time I use the Portable as it allows me to quickly switch between versions for testing, on this 32 bit device the permanent install is LAST pre-release (3.5.15481) has been so for months, and without significant crashes/problems (a few known oddities), but certainly not as you describe.

There should be no "switches" to overcome your issue , since it should not be misbehaving that way. When I WAS using 3.2 daily yours was not a common issue. 

The main problem with launching PDF  readers is that MS fiddled with the way they are used, when enforcing Edge as the default, so unknown if your mix of installed readers causes a knock on effect or a glitch in File Type Association. perhaps compare with an FTA tool with another system.

It sounds like a registry setting has been switched somehow to "all files" when it should be just the one. e.g. `SumatraPDF *` when it should be more like `SumatraPDF %1 %*`
